lib/src/widget/pin_widget.dart

```diff
@@ -208,6 +214,10 @@ class _PinInputTextFieldState extends State<PinInputTextField> {
         /// Hide the Cursor
         showCursor: false,
 
+        /// Autofill hints for the underlying platform.
+        autofillHints:
+            widget.enableCodeAutofill ? [AutofillHints.oneTimeCode] : null,
```

Comment:
my concern is we should not introduce new field to default define the behaviour, we can use `factory` to control it instead of this approach

Reply:
This isn't quite clear to me.

Do you mean we should allow the user to pass `autofillHints` directly in the constructor rather than controlling it with a boolean?

Reply:
yup. the idea was to give control by lib user instead of default behavior.